linters/osv-scanner/osv_to_sarif.py

```diff
@@ -86,17 +87,19 @@ def main(argv):
                     message = vuln["summary"]
                 else:
                     message = vuln["details"]
+
+                # Put single quotes around package names to clarify that it's a package
+                message = re.sub(f'({pkg["name"]})', r"'\1'", message)
+
                 description = (
                     f'Vulnerability in {pkg["name"]}@{pkg["version"]}: {message}'
                 )
 
-                lines = [
-                    zero_indexed + 1
-                    for zero_indexed, line in lockfile_lines
-                    if pkg["name"] in line and pkg["version"] in line
-                ]
```

Comment:
note: this list comprehension logic is more lines, more confusing, and less optimal, than just writing it as a regular for loop 😉